setup.py

```diff
@@ -2,9 +2,13 @@
     import regex as re
 except ImportError:
     import re
-from setuptools import find_packages, setup
+from setuptools import setup  # , find_packages  # unused
 
-__version__ ,= re.findall('__version__ = "(.*)"', open('lark/__init__.py').read())
+from lark import __version__
```

Comment:
I am pretty sure the goal was to *not* import `lark`

Reply:
Why though?

Reply:
Heh I'm late to the party. See my comments.